content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -194,6 +194,40 @@ data:
 EOF
 {{< /text >}}
 
+For clusters that use CoreDNS (> 1.4.0):
```

Comment:
Please implement this in-line YAML code using the snippets mechanism so we can implement automated samples testing. The way to do that is described here: https://istio.io/about/contribute/creating-and-editing-pages/#snippets

Thanks!

Reply:
Where do I place the snippet file (I couldn't find any reference) ?

Reply:
ping @adammil2000 could you tell me where I should put the files ?